### ISSUE TITLE
program path in hoobs.service is wrong

### ISSUE THREAD

**Issue by maximouche** [hoobs-org/HOOBS#318]
Context: I installed hoobs from the install.sh script in a centos 7 LXC container.

In the service file hoobs.service the programme to start is /usr/bin/hoobs.
This file doesn't exist on the system.
It is actually install in /usr/local/bin/hoobs

**Comment by mkellsy** [comment]
How did you install Node?

RHEL from the repo installs node in /usr/bin
If you use the Node repos, it is installed in /usr/local/bin

**Comment by mkellsy** [comment]
NEXT STEP
Need to detect where Node is installed.

**Comment by maximouche** [comment]
I used the install.sh script and the procedure described in the wiki for fedora : 
```
wget -q -O - https://raw.githubusercontent.com/hoobs-org/HOOBS/master/install.sh | sudo bash -
```

**Comment by mkellsy** [comment]
I am looking into the script, need to determine where node is installing. RHEL might be in /usr/local and Fedora is in /usr.

I wish RedHat would just accept that Node is supposed to be installed in /usr/local.

**Comment by mkellsy** [comment]
This now finds the NPM prefix location and sets the execution path properly.

Fixed in 3.1.2
